service/src/main/java/at/jku/isse/ecco/service/EccoService.java

```diff
@@ -53,10 +53,23 @@
 /**
  * A service class that gives access to high level operations like init, fork, pull, push, etc.
  */
+@Service
```

Comment:
adding springboot to a core-dependency that itself does not need it is not a good practice